### PR TITLE
Added instructions to enable sending of Devtools usage telemetry to K…

### PR DESCRIPTION
**Pull request by WayneFerrao**
## Description

This PR adds some documentation to the README to help devs get their local usage Telemetry sent to Kusto.

### CODE REVIEW

File: packages/tools/devtools/devtools-browser-extension/README.md [hunk 1]
```diff
@@ -106,6 +106,19 @@ To use a local build of this extension in your browser:
       In File Explorer or any other Windows application that can browse files, navigate to the path: \\wsl$.
     - If you are working in a [Codespace](https://code.visualstudio.com/docs/remote/codespaces) with Visual Studio Code, you can download the build artifacts by right-clicking on `dist/bundle` in the `Explorer` view and clicking `download`. This will download the files to your local machine, which you can upload to the browser.
 
+#### Sending local usage data to Kusto
+
+When using the Devtools browser extension, usage telemetry can be optionally generated and sent to Kusto. To do so, follow these instructions.
+
+1. Create a .env file in the devtools-browser-extension's root folder.
+2. Add the telemetry ingestion key to the env file. Consult Alejandro/Wayne to receive this key.
```
Comment (alexvy86) [inline packages/tools/devtools/devtools-browser-extension/README.md:114]: Would be nice to add here what the file contents need to look like, with a placeholder for the actual key.

File: packages/tools/devtools/devtools-browser-extension/README.md [hunk 2]
```diff
@@ -106,6 +106,19 @@ To use a local build of this extension in your browser:
...
+3. Run `pnpm run build` to build the extension.
+4. Load the unpacked extension in the browser by following the instructions above.
+5. When using the extension on the Devtools example app, ensure that Send Usage Telemetry is toggled in Settings.
+6. After using the extension, go to the Office Fluid Test database in Kusto & set your queries to the `office_fluid_devtools_generic` table.
```
Comment (alexvy86) [inline packages/tools/devtools/devtools-browser-extension/README.md:118]:
```suggestion
6. After using the extension, go to the Office Fluid Test database in Kusto and query the `office_fluid_devtools_generic` table.
```

**Review by alexvy86 — APPROVED** [review]
A few suggestions below, then 🚀

File: packages/tools/devtools/devtools-browser-extension/README.md (hunk 1 above)
Comment (Josmithr) [reply packages/tools/devtools/devtools-browser-extension/README.md:114]: Plus instructions for how to acquire that key (even if it's a link to internal-only documentation). And if it's only possible for internal devs to do this, we should note that as well.

**Pull request merged**